### DISCUSSION TITLE
Deploy this to Heroku

### DISCUSSION THREAD

**Discussion by ajmal-parta** [fastapi/full-stack-fastapi-template#2134] [Questions] [answered]
Would be nice to either have a heroku deployment YAML or instructions on how to properly deploy this repo to Heroku, same goes for suggestions on how to integrate this to be production ready on AWS.

**Comment by abrichr** [comment]
See https://github.com/tiangolo/full-stack-fastapi-postgresql/issues/322 for AWS deployment

**Comment by tiangolo** [comment, marked as answer]
As I wasn't happy with the product offerings to deploy apps, nor with the state of self hosting with Kubernetes, I started FastAPI Cloud ( https://fastapicloud.com/ ). At some point soon this project will have also the setup to deploy to FastAPI Cloud (although it's just one command, `fastapi deploy` :sweat_smile: ).
